docs/ManualDeploymentGuideScalarDLonAWS.md

```diff
@@ -0,0 +1,292 @@
+
+### Increase the number of Envoy Pods
+
+You can increase the number of envoy pods based on your requirements. The following steps will help you to achieve it:
+
+In `scalardl-custom-values.yaml`, you can update the number of `replicaCount` to the desired number of pod
+
+edit `scalardl-custom-values.yaml`
+
+```
+envoy:
+  replicaCount: 6
+```
+
+The number of deployable pods depends on the number of available nodes. So, you may need to increase the number of nodes from the AWS console.
+
+### Increase the resource of Envoy Pods
+
+You can increase the resource of envoy pods based on your requirements. The following steps will help you to achieve it:
+
+In `scalardl-custom-values.yaml`, you can update resource as follow
+
+edit `scalardl-custom-values.yaml`
+
+```
+envoy:
+  resources:
+    requests:
+      cpu: 400m
+      memory: 256Mi
+    limits:
+      cpu: 500m
+      memory: 328Mi
+```
+More information can be found in [the official documentation](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-requests-and-limits-of-pod-and-container)
+
+### Expose Envoy endpoint to public
+
+Scalar DL supports both public and private endpoints. The public endpoint allows you to access Scalar DL from outside the network so you can deploy your application outside the Scalar DL network. 
+The private endpoint allows you to access Scalar DL from within the network so you can deploy your application within the Scalar DL network or you can connect another VPC to the Scalar DL network using VPC peering.
+
+In `scalardl-custom-values.yaml`, you can update the `aws-load-balancer-internal` as false to expose `Envoy`
+
+```
+envoy:
+  service:
+    type: LoadBalancer
+    annotations:
+      service.beta.kubernetes.io/aws-load-balancer-internal: "false"
+```
+
+### Increase the number of Ledger Pods
+
+You can increase the number of ledger pods based on your requirements. The following steps will help you to achieve it:
+
+In `scalardl-custom-values.yaml`, you can update `replicaCount` to the desired number of pods.
+
+edit `scalardl-custom-values.yaml`
+
+```
+ledger:
+  replicaCount: 6
+```
+
+The number of deployable pods depends on the number of available nodes. You may need to increase the number of nodes from the AWS console.
+
+### Increase the resource of Ledger Pods
+
+You can increase the resource of ledger pods based on your requirements. The following steps will help you to achieve it:
+
+In `scalardl-custom-values.yaml`, you can update resource as follow
+
+edit `scalardl-custom-values.yaml`
+
+```
+ledger:
+  resources:
+    requests:
+      cpu: 1500m
+      memory: 2Gi
+    limits:
+      cpu: 1600m
+      memory: 4Gi
+```
+More information can be found in the [official documentation](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-requests-and-limits-of-pod-and-container)
+
+## Uninstall Scalar DL
```

Comment:
Why uninstalling only pods? How about other resources?

Reply:
> How about other resources?

I think it will be known to users, should we add that?.

Reply:
What do you mean by this `it will be known to users,`. I don't get it.

Reply:
> What do you mean by this it will be known to users,. I don't get it.

We expect this guide will be used by people with basic knowledge of AWS and EKS, so we expect they will know how to remove other resources

Reply:
I meant, you show how to setup and install other resources but show only uninstall Scalar DL pods, isn't this inconsistent?
For example, why no section for removing k8s cluster?